### PR TITLE
Replace var to let and remove unecessary else

### PR DESCRIPTION
**Pull request by leonascimento**

### CODE REVIEW

File: dist/highcharts-layout/js/message.js [hunk 1]
```diff
@@ -3,20 +3,19 @@ const hcUtils = {
     parseOptions: function (chartOptions) {
         const parseFunction = this.parseFunction;
 
-        var options = JSON.parse(chartOptions, function (val, key) {
+        let options = JSON.parse(chartOptions, function (val, key) {
```
Comment (luisfmsouza) [inline dist/highcharts-layout/js/message.js]: Why not `const`? Here and in the other places.

Comment (leonascimento) [reply dist/highcharts-layout/js/message.js]: Great, it makes total sense I forgot to think that in this context we are not assigning this variable again.

Comment (sebastianbochan) [reply dist/highcharts-layout/js/message.js]: Thank you for the suggestion, will fix also.

**Comment by sebastianbochan** [comment]
Closed by commit: https://github.com/highcharts/highcharts-react-native/commit/465753d6426b832398288c7f43fa79ce24f44755

**Pull request closed without merging**